src/main/java/frc/trigon/robot/utilities/JsonHandler.java

```diff
@@ -0,0 +1,71 @@
+package frc.trigon.robot.utilities;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import edu.wpi.first.wpilibj.Filesystem;
+
+import java.io.*;
+
+public class JsonHandler {
+    private static final Gson gson = new GsonBuilder().setPrettyPrinting().create();
+    private static final String path = Filesystem.getDeployDirectory().getPath() + "\\";
+
+    /**
+     * Parses the given object to a new or existing JSON file,
+     * using a safe way of saving. (creating a temporary file and then
+     * replacing the original file with the temporary one)
+     *
+     * @param object the object to save
+     * @param name   the name of the file to save to or create
+     */
+    public static void saveToJsonFile(Object object, String name) throws IOException {
```

Comment:
Don't use the word "save". "write" maybe.
And in the function name you need to explain that you're not getting text, but an object that you later parse to json
So maybe `parseToJsonAndWrite`